draft-keytrans-mcmillion-protocol.md

```diff
@@ -78,10 +78,23 @@ properties are maintained.
 
 # Tree Construction
 
-KT relies on two combined hash tree structures: log trees and prefix trees. This
-section describes the operation of both at a high level and the way that they're
-combined. More precise algorithms for computing the intermediate and root values
-of the trees are given in {{cryptographic-computations}}.
+KT allows clients of a service to query the keys of other clients of the same
+service. To do so, KT maintains two types of maps: (i) one associates *version
```

Comment:
Maintains two things:
- A log of changes to lookup keys
- A set of key-version pairs